### PR TITLE
Update prefs.ini with new flash-blocking prefs

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Comment by rbillings** [comment]
This matches all of my notes on the changes. R+

File: shavar/e2e-test/prefs.ini [hunk 1]
```diff
@@ -9,20 +9,28 @@ browser.safebrowsing.phishing.enabled = false
 browser.safebrowsing.malware.enabled = false
 privacy.trackingprotection.enabled = true
 browser.safebrowsing.provider.mozilla.nextupdatetime = 1
-browser.safebrowsing.provider.mozilla.gethashURL = https://shavar.stage.mozaws.net/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.gethashURL = http://127.0.0.1
```
Comment (rpappalax) [inline shavar/e2e-test/prefs.ini:12]: not sure what all the prefs are used for but is there a reason this one would be set to localhost (vs. stage?)

Comment (rbillings) [reply shavar/e2e-test/prefs.ini:12]: My understanding is that the environments listed below do not refer to this gethashURL, so there was no need for it to be the stage environment.

**Review by rbillings — APPROVED** [review]

**Comment by rbillings** [comment]
I'm going to merge this in the interest of getting it updated to complete testing asap. We can re-open if an issue is found.

File: shavar/e2e-test/prefs.ini (hunk 1 above)
Comment (rpappalax) [reply shavar/e2e-test/prefs.ini:12]: @fmarier @groovecoder  
ah, ok.  So what is gethashURL used for then and do we need to set it at all when testing against stage (or can we leave the default value in place)?

Comment (fmarier) [reply shavar/e2e-test/prefs.ini:12]: Gethash is unused. If a list is mentioned in `urlclassifier.disallow_completions` then we will not use gethash.

You're right, we can probably just remove that line and just keep the default value.

**Pull request merged**